charts/lagoon-logs-concentrator/Chart.yaml

```diff
@@ -19,7 +19,7 @@ type: application
 # time you make changes to the chart and its templates, including the app
 # version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.47.0
+version: 0.48.0
 
 # This section is used to collect a changelog for artifacthub.io
```

Comment:
The CI failure is due to this field not being updated. This field must be updated on every PR because it generates the changelog on artifacthub. The format is documented [here](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations).